devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -16,8 +16,8 @@
         <quarkus.platform.artifact-id>{quarkus.platform.artifact-id}</quarkus.platform.artifact-id>
         <quarkus.platform.version>{quarkus.platform.version}</quarkus.platform.version>
         <quarkus-plugin.version>{quarkus.plugin.version}</quarkus-plugin.version>
-        <compiler-plugin.version>{maven.version.maven-compiler-plugin}</compiler-plugin.version>
-        <surefire-plugin.version>{maven.version.maven-surefire-plugin}</surefire-plugin.version>
+        <compiler-plugin.version>{version.maven-compiler-plugin}</compiler-plugin.version>
```

Comment:
why are we doing two different naming strategies here ? 

it should be `maven-compiler.plugin.version` to match what we do everywhere else (except code.quarkus for some reason :)